src/main/java/br/com/senior/core/tenant/TenantClient.java

```diff
@@ -49,23 +49,23 @@ public TenantClient(String token, Environment env) {
     /**
      * Obtêm os dados de um tenant
      *
-     * @param tenantName - Nome do tenant
+     * @param input - Payload de entrada
      * @return - Os dados do tenant
      * @throws ServiceException - Erro tratado de serviço
      */
-    public TenantOutput getTenantByName(String tenantName) throws ServiceException {
-        return execute(getQueriesUrl(EndpointPath.Tenant.GET_TENANT_BY_NAME), new GetTenantByNameInput(tenantName), this.token, TenantOutput.class);
```

Comment:
Aqui eu acho q deveríamos manter a opção de apenas passar a String do tenant.
Podemos deixar os 2, mas penso que apenas com a String, facilitamos a vida de que for utilizar este client.